src/MLCompute/MLHelpers.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Runtime.InteropServices;
+
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+namespace MLCompute {
+
+	[iOS (14,0)][TV (14,0)][Mac (10,16)]
+	[NoWatch]
+	public static class MLCActivationTypeExtensions {
```

Comment:
Is `MLC` a 3 letter acronym here?

Reply:
yes, `MLC` -> `MLCompute` -> `Machine Learning Compute`
However we don't PascalCase Apple's framework prefixes, e.g. `MPS` for Metal Performance Shaders
I don't recall all reasons but it's a pain to use `Name = "x"` in every type's bindings because none would register properly with ObjC

Reply:
Thank you for the explanation! 3 letter acronyms might be my ☠️